sys/page_revisions.js

```diff
@@ -83,15 +83,7 @@ class PRS {
                 { attribute: 'latest_rev', type: 'static' },
                 { attribute: 'tid', type: 'range', order: 'desc' },
                 { attribute: 'page_deleted', type: 'static' }
-            ],
-            secondaryIndexes: {
-                by_rev: [
-                    { attribute: 'rev', type: 'hash' },
-                    { attribute: 'tid', type: 'range', order: 'desc' },
-                    { attribute: 'title', type: 'range', order: 'asc' },
-                    { attribute: 'restrictions', type: 'proj' }
-                ]
-            }
+            ]
```

Comment:
Should we also bump the version since it has changed?

Reply:
Did bump the version. We should not forget that the storage module doesn't actually delete the tables after deleting the index, so we will need to drop key spaces manually.